ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessActivityResource.scala

```diff
@@ -73,8 +73,8 @@ class AttachmentResources(attachmentService: ProcessAttachmentService,
           }
         }
       }
-    } ~ path("processes" / Segment / LongNumber / "activity" / "attachments" / LongNumber) { (processName, versionId, attachmentId) =>
-      (get & processId(processName)) { processId =>
+    } ~ path("processes" / Segment / VersionIdSegment / "activity" / "attachments" / LongNumber) { (processName, versionId, attachmentId) =>
+      (get & processId(processName)) { _ =>
```

Comment:
I wonder, what do we need processName/versionId for?? it's also a security problem - you can pass attachmendId from scenario you don't have access to, processName/version you have access to, and.. 
let's add FIXME for that...

Reply:
Yep, a have looked on it. But if we want use `Mapped To [Long]`, we provide another dependency to api package. I'm not sure do we want it?